js/articles.js

```diff
@@ -0,0 +1,71 @@
+fetch('jsons/articles.json')
+  .then(response => {
+    return response.json()
+  })
+  .then(data => {
+    // Work with JSON data here
+    articlesCounter = data["articles"].length;
+    let a = Math.floor(Math.random() * articlesCounter);
+    let b = Math.floor(Math.random() * articlesCounter);
+    let c = Math.floor(Math.random() * articlesCounter);
+
+    const width = screen.width;
+    const articles = document.getElementsByClassName("articles")[0];
+    if (width>=800){
+      bildArticle(a);
+      bildArticle(b);
+      bildArticle(c);
+    } else if(width<400){
+      bildArticle(a);
+    } else{
+      bildArticle(a);
+      bildArticle(b);
+    }
+
+    function bildArticle(i){
+      let articlesConteiner = document.createElement("div");
+      articlesConteiner.classList.add("articles-conteiner");
+
+      let imgLink = document.createElement("a");
+      imgLink.href = "#article$"+data["articles"][i]["id"];
```

Comment:
articles - will be separate page with separate html file which will caled 'article.html'
so you should create real link
it will look like
"somehost.com/articles.html?3'
where 3 will be param id
this information are available if you get the location
try console.log(location) - it's global object
and there will be property search - here will stored the id.

your code is going to redirect user on THIS page but inanother section with name=article